### ISSUE TITLE
Pagination of texts in coursepack as selection

### ISSUE THREAD

**Issue by tonyahowe** [LiteratureInContext/LiC-app#340]
It looks like text excerpts that are added to coursepacks aren't paginated, though pagination is in the file. On both dev and prod. This screenshot is of a coursepack selection from Robinson Crusoe:
![Screenshot 2024-08-07 at 4 25 49 PM](https://github.com/user-attachments/assets/908685a0-2119-416a-b347-47e04e5dd7e9)

**Comment by wsalesky** [comment]
@tonyahowe pagination is still in development, so it will need to be applied to coursepacks once we have it working correctly on the dev site.

**Comment by tonyahowe** [comment]
Cool thanks!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by wsalesky** [comment]
@tonyahowe Perhaps we can discuss in meeting, would be easier to redirect to the work, or use an iframe, rather than to actually load all the full works in the coursepack HTML display. They will still need to be loaded in the PDF and EPUB versions of course.

**Comment by tonyahowe** [comment]
Yes, let's talk about it. I think the excerpts should still be in the CP, but I'm curious what others think. Thanks!
